compositional_skills/extraction/it_tickets/qna.yaml

```diff
@@ -0,0 +1,104 @@
+task_description: Metadata identification from an IT incidence ticket and generating and output in json format
+created_by: Paula Rodriguez
```

Comment:
This needs to be your GitHub username :)

Reply:
Done! Thanks for the reminder :)

Reply:
Ah sorry, your actual GitHub username, not your name. So `paulaviguri`